src/fs/protocol/private/mmpt.ts

```diff
@@ -0,0 +1,100 @@
+import { AddResult, CID } from "../../../ipfs"
+import * as basic from '../basic'
+import * as link from '../../link'
+import { Links } from '../../types'
+
+const nibbles = { "0": true, "1": true, "2": true, "3": true, "4": true, "5": true, "6": true, "7": true,
+                  "8": true, "9": true, "a": true, "b": true, "c": true, "d": true, "e": true, "f": true,
+                } as {[key: string]: boolean}
```

Comment:
Wait, what is this for? Why all true?

Reply:
Oh testing membership? Why not array membership? Speed?

Reply:
yup for testing membership. I just figured this was faster than looping an array :man_shrugging: